tools/validators/ontology_validator/yamlformat/validator/field_lib.py

```diff
@@ -80,7 +80,15 @@ def IsFieldDefined(self, fieldname, namespace_name):
     """
     return fieldname in self._namespace_map.get(namespace_name, set())
 
+  def GetFieldsMap(self, namespace_name):
+    """Returns the fields defined within namespace.
```

Comment:
be more specific about _exactly_ what is returned, including exactly what the keys are expected to be in the map. (because python)